### DISCUSSION TITLE
Asked about the Inbound Interconnection part on the Configuration Examples page via email.

### DISCUSSION THREAD

**Discussion by JohnDavid9109** [hnimminh/libresbc#107] [Q&A]
![image](https://github.com/hnimminh/libresbc/assets/141992286/dcfb4f7b-6ce7-49c6-8f61-9399ab247008)

What is value_error ?

**Comment by hnimminh** [comment]
@JohnDavid9109  The error show that you did not created **capacity** class & **media** class. 

Please create classes before use them.

**Reply by JohnDavid9109** [reply]
I already create classes. I found the reason, it was because the class name was different from what I had already created. I changed it to the following and it worked fine. I think it probably needs to be modified on the Example page as well.
pstn -> pcm
smb -> small

**Reply by hnimminh** [reply]
can you detailize what wrong with example? @JohnDavid9109 
As i understand: you create class --> you change name of them --> then you config interconnect that refer to classes which is not existing.

**Reply by JohnDavid9109** [reply]
In Configuration Examples, we named the Media Class as "name": "pcm", but when creating Inbound Interconnection, we call it as "media_class": "pstn". Similarly for Capacity Class, make it "name": "small" and call it "capacity_class": "smb".

**Reply by hnimminh** [reply]
Thanks for letting me know, I will correct that soon
